de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5527630_sys_gh5386_CreateJsonReportsUserAndHisAuthToken.sql

```diff
@@ -0,0 +1,26 @@
+-- create user Json Reports
+
+-- 2019-07-19T09:29:11.356Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_User (AD_Client_ID,AD_Language,AD_Org_ID,AD_User_ID,Comments,Created,CreatedBy,Description,Fresh_Gift,IsAccountLocked,IsActive,IsBillToContact_Default,IsDecider,IsDefaultContact,IsFullBPAccess,IsInPayroll,IsLoginAsHostKey,IsManagement,IsMFProcurementUser,IsMultiplier,IsSystemUser,Login,LoginFailureCount,Name,NotificationType,Processing,UnlockAccount,Updated,UpdatedBy,Value) VALUES (1000000,'en_US',1000000,540057,'',TO_TIMESTAMP('2019-07-19 12:29:11','YYYY-MM-DD HH24:MI:SS'),100,'User used for authentication for Json Reports','N','N','Y','N','N','N','Y','N','N','N','N','N','Y','Json Reports',0,'Json Reports','N','N','N',TO_TIMESTAMP('2019-07-19 12:29:11','YYYY-MM-DD HH24:MI:SS'),100,'jreports')
```

Comment:
does it need to be a "SystemUser=Y" to work? Asking because i think we should make sure that one can't actually log into the webUI with this user